### PR TITLE
lookup client & objstore

### PR DESCRIPTION
**Pull request by jeffpc**
Still need a objstore backend support.

### CODE REVIEW

File: src/objstore/include/nomad/objstore.h [hunk 1]
```diff
@@ -83,5 +83,7 @@ extern int objstore_getroot(struct objstore *vg, struct nobjhndl *hndl);
 /* object operations */
 extern int objstore_getattr(struct objstore *vg, const struct nobjhndl *hndl,
 			    struct nattr *attr);
+extern int objstore_lookup(struct objstore *vg, const struct nobjhndl *dir,
+			   const char *name, struct nobjhndl *child);
```
Comment (Thynix) [inline src/objstore/include/nomad/objstore.h:87]: Alignment looks like it's missing a space here.

File: src/objstore/include/nomad/objstore_impl.h [hunk 2]
```diff
@@ -70,5 +73,8 @@ extern int objstore_vol_getroot(struct objstore_vol *vol, struct nobjhndl *hndl)
 /* wrappers for object ops */
 extern int objstore_vol_getattr(struct objstore_vol *vol,
 				const struct nobjhndl *hndl, struct nattr *attr);
+extern int objstore_vol_lookup(struct objstore_vol *vol,
+			       const struct nobjhndl *dir, const char *name,
```
Comment (Thynix) [inline src/objstore/include/nomad/objstore_impl.h:77]: Alignment missing spaces here too.

File: src/objstore/vg.c [hunk 3]
```diff
@@ -140,3 +140,28 @@ int objstore_getattr(struct objstore *vg, const struct nobjhndl *hndl,
 
 	return ret;
 }
+
+int objstore_lookup(struct objstore *vg, const struct nobjhndl *dir,
+		    const char *name, struct nobjhndl *child)
```
Comment (Thynix) [inline src/objstore/vg.c:145]: Missing space here.

**Comment by Thynix** [comment]
This looks like it's missing something to actually perform lookup - that looks intended.

Other than that and the things I mentioned this LGTM.

File: src/objstore/include/nomad/objstore.h (hunk 1 above)
Comment (jeffpc) [inline src/objstore/include/nomad/objstore.h:87]: It's just github misrendering things... :/

Comment (kepish) [inline src/objstore/include/nomad/objstore.h:87]: I don't think it is; I think he used tabs for alignment. Bad Jeff!

**Comment by jeffpc** [comment]
Ok, I replaced tabs with spaces.  I'll push it.

**Pull request closed without merging**